.github/ISSUE_TEMPLATE/challenge_proposal.md

```diff
@@ -0,0 +1,20 @@
+---
+name: 🚀 Challenge Proposal
+about: I want to add a new challenge!
+title: ''
+labels: challenge
+assignees: ''
+
+---
+
+## Describe the challenge you'd like to add
+A clear and concise description of what you want to happen. Add any considered drawbacks.
+
+## Suggest a name
+The name of your baby. Consider that there might be adjustment in the challenge name.
```

Comment:
I think this one is better `Consider that there might be adjustments in the challenge name.` We may need to make more than just one adjustment.